content/en/docs/security/best-practices.md

```diff
@@ -0,0 +1,246 @@
+---
+title: "Security Best Practices"
+linkTitle: "Best Practices"
+description: "Best practices for securing Flux deployments."
+weight: 140
+---
+
+# Flux Security Best Practices
+
```

Comment:
![DeepinScreenshot_select-area_20220719131234](https://user-images.githubusercontent.com/387444/179737040-310758b0-9229-4a3b-85f8-00bf18871654.png)
```suggestion
```
This avoids the duplicate header.